server.js

```diff
@@ -0,0 +1,41 @@
+require('dotenv').config()
```

Comment:
I think we have similar file called index.js from where we start our api - not to this line but to whole server.js